src/data/services/EnterpriseAccessApiService.js

```diff
@@ -147,15 +147,21 @@ class EnterpriseAccessApiService {
    * List content assignments for a specific AssignmentConfiguration.
    */
   static listContentAssignments(assignmentConfigurationUUID, options = {}) {
-    const params = new URLSearchParams({
+    const params = {
       page: 1,
       page_size: 25,
       // Only include assignments with allocated or errored states. The table should NOT
       // include assignments in the canceled or accepted states.
       state__in: 'allocated,errored',
       ...snakeCaseObject(options),
-    });
-    const url = `${EnterpriseAccessApiService.baseUrl}/assignment-configurations/${assignmentConfigurationUUID}/admin/assignments/?${params.toString()}`;
+    };
+    // Rename the `learner_state` filter to `learner_state__in`
+    if (params.learner_state) {
+      params.learner_state__in = params.learner_state;
+      delete params.learner_state;
+    }
```

Comment:
[suggestion] May be a bit cleaner along the lines of:

```jsx
const { learnerState, ...optionsRest } = options;
const params = new URLSearchParams({
  // ... other attrs
  learner_state__in: learnerState,
  ...snakeCaseObject(optionsRest),
});
```

Reply:
D'oh: needs a revision... take two:

```jsx
const { learnerState, ...optionsRest } = options;
const params = new URLSearchParams({
  // ... other attrs
  ...snakeCaseObject(optionsRest),
});
if (learnerState) {
    params.learner_state__in = learnerState;
}
```

Reply:
Basically just to avoid the `delete` lol. Long-winded way to say that haha, oops 😄